apps/map/codegen.yml

```diff
@@ -0,0 +1,61 @@
+overwrite: true
+schema:
+  - "http://localhost:4455/graphql"
```

Comment:
I think it make more sense to use the locally save schema.graphql like we do here https://github.com/GaloyMoney/galoy/blob/main/apps/admin-panel/codegen.yml#L3